docs/section5/Rule5-48.md

```diff
@@ -15,37 +15,26 @@
 **Data Lookup:** None  
 **Function Call:** 
 
-  1. get_zone_conditioning_category()
-  2. match_data_element()
+  1. match_data_element()
 
```

Comment:
We do need get_zone_conditioning_category per above.

Reply:
This function is now added.